### ISSUE TITLE
Documentation of supported browsers

### ISSUE THREAD

**Issue by ipoint-pgerhard** [surveyjs/survey-creator#1482]
### Are you requesting a feature, reporting a bug or ask a question?

Question:

Back in 2019 the question was asked if there is documentation available on the supported browsers ([#497](https://github.com/surveyjs/survey-creator/issues/497)). I am now running into an issue with IE 11 and wondering the same thing. Is there documentation available on which browsers are supported?

**Comment by dmitrykurmanov** [comment]
For the creator(v1) we supported IE11. Could you please describe your issue?

**Comment by ipoint-pgerhard** [comment]
We are embedding the SurveyJS editor inside a Vaadin 7 (https://vaadin.com/) application using a Javascript Component (https://vaadin.com/docs/v7/framework/gwt/gwt-javascript). Attempting to display this component inside IE11 fails. However this could be a number of things, we are currently trying to pinpoint the error and I wanted to be able to exclude SurveyJS. 

Is there documentation available on which browsers are supported? Just to save others from having to ask a similar question at more or less regular intervals.

**Comment by andrewtelnov** [comment]
@ipoint-pgerhard Indeed, discuss it today on stand-up meeting. It makes sense to create this page.
We are supporting: Chrome, Edge, Safari and FireFox.
Regarding IE11, Creator V1 should support it. We will need to check it again. We decided do not support IE11 in V2 since it requires a lot of recourses to support IE11 without visible advantages.

I have just try our [creator](https://surveyjs.io/create-survey) on IE11 (20H2) and seems to work.

Thank you,
Andrew
